hack/make/build-deb

```diff
@@ -86,6 +86,16 @@ set -e
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
 			RUN cp -aL hack/make/.build-deb debian
 			RUN { echo '$debSource (${debVersion}-0~${suite}) $suite; urgency=low'; echo; echo '  * Version: $VERSION'; echo; echo " -- $debMaintainer  $debDate"; } > debian/changelog && cat >&2 debian/changelog
+		EOF
+		# Remove the following case-based substitution when none of these are supported, and the TasksMax value is uncommented in docker.service
+		case "$version" in
+		debian-jessie|debian-wheezy|ubuntu-precise|ubuntu-trusty|ubuntu-wily)
+			;;
+		*)
+			echo "RUN sed -i -- 's/#TasksMax=infinity/TasksMax=infinity/' contrib/init/systemd/docker.service" >> "$DEST/$version/Dockerfile.build"
```

Comment:
Sorry for one more message, but I finally got the build-deb script to finish without crashing b/c of some flaky repo, and it did build docker-engine_1.12.0~rc1-0~stretch_amd64.deb. 

That said, do what you think makes the most sense. But for some reason it's only affecting the raspbian builds.